nginx.conf

```diff
@@ -0,0 +1,9 @@
+server {
+        listen 80;
+        server_name localhost;
+        root /usr/share/nginx/html;
+        index index.html;
+        location / {
+            try_files $uri $uri/ /index.html =404;
```

Comment:
this is unsafe solution
if your codebase contain sensitive files - you allow end users to render them, i.e. .env type of files.
check official docs what try_files does.

https://nginx.org/en/docs/http/ngx_http_core_module.html#try_files

find a better and safer way for your particular case.

Reply:
Don't see that it was fixed